.github/workflows/backend.yml

```diff
@@ -3,31 +3,77 @@ on:
   pull_request:
     types: ['opened', 'edited', 'reopened', 'synchronize']
     paths:
-    - 'kafka-ui-api/**'
+      - 'kafka-ui-api/**'
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - name: Cache local Maven repository
-      uses: actions/cache@v2
-      with:
-        path: ~/.m2/repository
-        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-        restore-keys: |
-          ${{ runner.os }}-maven-
-    - uses: actions/checkout@v2
-    - name: Set up JDK 1.13
-      uses: actions/setup-java@v1
-      with:
-        java-version: 1.13
-    - name: Cache SonarCloud packages
-      uses: actions/cache@v1
-      with:
-        path: ~/.sonar/cache
-        key: ${{ runner.os }}-sonar
-        restore-keys: ${{ runner.os }}-sonar
-    - name: Build and analyze
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
-        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-      run: mvn -B verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - uses: actions/checkout@v2
+      - name: Set the value1
+        id: step_one
+        run: |
+          echo "USE_LOCAL_BROWSER=false" >> "./kafka-ui-e2e-checks/.env"
+      - name: Set the value2
```

Comment:
can we name the steps  with something like 'setting .env' /etc, just to clarify steps? Also we probably can do them in on single step